Apply two small documentation improvements

Stephan202 opened the pull request.

mernst commented:
@Stephan202 Sorry for the delay in responding; I was away from email.
Could you please add your name to doc/manual/contributors.tex in order to fix the continuous integration failure?
Also, do you approve of relicensing your contributions to the Chejcker Framework (currently GPL2) to a more liberal license such as the MIT License?
Thanks!

Stephan202 commented:
Hi @mernst, sorry for missing the build failures. I rebased the branch and squashed the `contributors.tex` change into the first commit. (That was faster than checking whether in this repo you merge or squash-and-rebase; I have limited computer access myself atm.)

I'm completely fine with a more liberal license. :+1:  (Most/all code we open source at Picnic is also MIT licensed.)

mernst commented:
Thanks very much for these fixes!  We appreciate it.
